server/app/models/ApplicationModel.java

```diff
@@ -177,6 +187,15 @@ void setLatestStatusForTest(String latestStatus) {
     this.latestStatus = latestStatus;
   }
 
+  /**
+   * The latest note is never directly set. It is only set using update query from the
+   * ApplicationEventRepository.
+   */
+  @VisibleForTesting
+  public void setLatestNote(String applicationNote) {
```

Comment:
**suggestion, blocking**: Two questions:
- Can we change this to something like `setLatestNoteForTest()` to make it clear it shouldn't be used?
- Does ErrorProne yell at your if you try to use this if you try to use it outside a test?

Reply:
Relatedly, should this be marked as `private` instead of `public`?

Reply:
Good catch! it should be protected. Same as the setLatestStatus() above it in the application model

Reply:
Since it is not visible outside the class being a protected member, none of the repositories or services can access it

Reply:
Also @avaleske, leaving it as protected, ErrorProne behaves well :)